README.MD

```diff
@@ -1,6 +1,6 @@
 # StorageOS kubectl plugin
 
-[![Go Report Card](https://goreportcard.com/badge/github.com/storageos/kubectl-storageos)](https://goreportcard.com/report/github.com/storageos/kubectl-storageos) [![Active](http://img.shields.io/badge/Status-Active-green.svg)](https://github.com/storageos/kubectl-storageos) [![PR's Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](https://github.com/storageos/kubectl-storageos/pulls) [![Build](https://github.com/storageos/kubectl-storageos/actions/workflows/kuttl-e2e-test.yaml/badge.svg)](https://travis-ci.com/storageos/kubectl-storageos) [![License](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
+[![Go Report Card](https://goreportcard.com/badge/github.com/storageos/kubectl-storageos)](https://goreportcard.com/report/github.com/storageos/kubectl-storageos) [![Build](https://github.com/storageos/kubectl-storageos/actions/workflows/kuttl-e2e-test.yaml/badge.svg)](https://travis-ci.com/storageos/kubectl-storageos) [![CodeQL](https://github.com/storageos/kubectl-storageos/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/storageos/kubectl-storageos/actions/workflows/codeql-analysis.yml) [![Active](http://img.shields.io/badge/Status-Active-green.svg)](https://github.com/storageos/kubectl-storageos) [![PR's Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](https://github.com/storageos/kubectl-storageos/pulls) [![License](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
```

Comment:
There seems to be the incorrect link for the `Build` badge. We don't use travisCI in this project as per my knowledge. And the badge is based on github actions but the link is to travisCI.